src/metadata.rs

```diff
@@ -84,6 +85,9 @@ impl Metadata {
     pub fn episode(&self) -> Option<i32> {
         self.episode
     }
+    pub fn last_episode(&self) -> Option<i32> {
```

Comment:
Probably merits some documentation of how and why that field exists for users.

Reply:
Yes. Of course.

I am not happy with the actual naming. Any thoughts.

Reply:
How about instead of this implied interval, a `fn multi_episode(&self) -> Option<Vec<i32>>`

Reply:
I will take a look. But I was actually thinking that it might not be needed.
Also perhaps just `fn episodes(&self) -> Option<Vec<i32>>` making it plural should be fairly clear. Though I guess typing mistakes could cause issues...

I will come up with some sample code when I have a moment.